### PR TITLE
Add initial ibu post upgrade validations structure

### PR DESCRIPTION
**Pull request by mcornea**

### CODE REVIEW

**Comment by mcornea** [comment]
@mpmaruthu Hey Peri o/ I added some initial post upgrade validation tests which could be shared between the various upgrade tests. Please review and let me know how the structure looks to you.

File: tests/imagebasedupgrade/validations/post_ibu_validations.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/openshift-kni/eco-goinfra/pkg/polarion"
+	"github.com/openshift-kni/eco-gosystem/tests/imagebasedupgrade/internal/imagebasedupgradeparams"
+)
+
+// PostUpgradeValidations is a dedicated func to run post upgrade test validations.
+func PostUpgradeValidations() {
+	Describe(
+		"PostIBUValidations",
+		Label("PostIBUValidations"), func() {
+			It("Validate Cluster version", polarion.ID("99999"), Label("ValidateClusterID"), func() {
```
Comment (mpmaruthu) [inline tests/imagebasedupgrade/validations/post_ibu_validations.go:15]: This test case validates "Cluster version" post upgrade . From my view, Label has a typo error - Should be ValidateClusterVersion

File: tests/imagebasedupgrade/validations/post_ibu_validations.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+				By("Validate cluster ID remains the same", func() {
```
Comment (mpmaruthu) [inline tests/imagebasedupgrade/validations/post_ibu_validations.go:16]: Test Step name change: "Validate cluster ID remains the same" => "Validate upgraded cluster version  report correct version"

**Review by mpmaruthu — COMMENTED** [review]
Minor comments from my view, otherwise looks good to me.

File: tests/imagebasedupgrade/validations/post_ibu_validations.go (hunk 1 above)
Comment (mcornea) [reply tests/imagebasedupgrade/validations/post_ibu_validations.go]: Thanks, updated.

File: tests/imagebasedupgrade/validations/post_ibu_validations.go (hunk 2 above)
Comment (mcornea) [reply tests/imagebasedupgrade/validations/post_ibu_validations.go]: Thanks, updated.

**Pull request merged**